src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs

```diff
@@ -221,7 +221,7 @@ public Color CalendarForeColor
                 if (!value.Equals(calendarForeColor))
                 {
                     calendarForeColor = value;
-                    SetControlColor(NativeMethods.MCSC_TEXT, value);
+                    SetControlColor((int)Interop.MonthCalendar.SizeColor.MCSC_TEXT, value);
```

Comment:
```suggestion
                    SetControlColor((int)Interop.ComCtl32.MCSC.TEXT, value);
```

Also the convention is to bring in static Interop so that this line would not need to include the `Interop.`